client/src/io/fs.rs

```diff
@@ -93,3 +93,92 @@ pub fn unzip(profile: &Profile) -> Result<()> {
 
     Ok(())
 }
+
+#[cfg(unix)]
```

Comment:
Would suggest to move nix specifics to `nix.rs` like [`windows.rs`](https://github.com/Songtronix/Airshipper/blob/staging/client/src/windows.rs) and cfg-gate it.